css/webstyle.css

```diff
@@ -0,0 +1,346 @@
+  color: #344563;
+  padding-right: 2%;
+  margin-block-end: 12px;
+}
+
+p {
+  font-size: 20px;
+  font-weight: 400;
+  line-height: 28px;
+}
+
+h4 {
+  margin-block-start: 0;
+  margin-block-end: 0;
+  color: #7f8cff;
+  font-weight: 500;
+  line-height: 24px;
+}
+
+.card {
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  width: 80.0277%;
+  align-self: center;
+  background-color: #fff;
+  margin: 9.97%;
+  margin-bottom: 0;
+  border: 1px solid #dfe1e6;
+  border-radius: 24px;
+}
+
+figure,
+div {
+  margin-inline: 4.4%;
+  margin-block: 4.4%;
+  display: flex;
+  align-items: center;
+}
+
+.reverse {
+  order: 1;
+}
+
+img {
+  width: 100%;
+}
+
+div {
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 2.8%;
+}
+
+h2 {
+  color: #091e42;
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 52px;
+  margin-block: 0;
+}
+
+.cardtxt {
+  font-size: 16px;
+  line-height: 24px;
+}
+
+.work li {
+  color: #344563;
+  font-size: 18px;
+  line-height: 24px;
+  font-weight: 600;
+  padding-right: 7%;
+}
+
+.work :nth-last-child(-n+2) {
+  color: #7a869a;
+  list-style: disc;
+  padding-left: 0;
+}
+
+ul .languages {
+  color: #6070ff;
+  background-color: #ebebff;
+  border-radius: 8px;
+  text-align: center;
+  padding: 4px 12px;
+  margin-bottom: 0;
+  display: block;
+}
+
+form {
+  align-self: stretch;
+  margin-bottom: 5%;
+  height: 100%;
+}
+
+#hand {
+  background-image: url(../images/hand-cur.png);
+  background-repeat: no-repeat;
+  background-position: 90% 20%;
+}
+
+button {
+  color: #396df2;
+  margin-top: 12px;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  text-align: center;
+  padding: 12px;
+  font-weight: 500;
+  width: 124px;
+  line-height: 24px;
+  font-size: 17px;
+  letter-spacing: 0.3px;
+  background-color: #fff;
+}
+
+.aboutme {
+  display: grid;
+  grid-template-rows: repeat(3, auto);
+  grid-template-columns: 42% 50%;
+  background-color: #fff;
+  border-top-right-radius: 8.5% 8.5%;
+  margin-top: 9.97%;
+  padding-top: 9.97%;
+}
+
+.abouttxt {
+  grid-row: 1;
+  grid-column: 1;
+  padding-left: 0;
+  margin-left: 24%;
+}
+
+#getbutton {
+  width: 162px;
+}
+
+#aboutlang {
+  grid-row: 1;
+  grid-column: 2;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+}
+
+h6 {
+  width: 100%;
+  font-size: 20px;
+  font-weight: 500;
+  line-height: 24px;
+  display: flex;
+  justify-content: space-between;
+  padding-right: 15px;
+  margin-block-end: 0;
+}
+
+#vertical {
+  writing-mode: vertical-rl;
+  text-orientation: mixed;
+  align-self: baseline;
+}
+
+span {
+  padding-right: 2%;
+}
+
+.langicons {
+  padding: 76px 12px 20px 4%;
+  margin-top: 8%;
+  width: 122px;
+  background-repeat: no-repeat;
+  background-color: #f7f7f9;
+  background-position: 12% 12%;
+  border-radius: 8px;
+}
+
+#js {
+  background-image: url(../images/JS.png);
+}
+
+#hm {
+  background-image: url(../images/HTML.png);
+}
+
+#cs {
+  background-image: url(../images/CSS.png);
+}
+
```

Comment:
- [ ] The language icons & also the display for that section are not correct.


- **Your project**

![your-aboute_me](https://user-images.githubusercontent.com/61562626/134772082-b0925452-426e-4746-b2bc-7d20153ddb5f.jpeg)

- **Figma template**

![about _me](https://user-images.githubusercontent.com/61562626/134772090-0599c31b-90b8-4eab-b73b-23dad668a577.jpeg)

Reply:
Screen shot from index.html